deploy/helm/sumologic/requirements.yaml

```diff
@@ -6,7 +6,7 @@ dependencies:
   - name: prometheus-operator
     version: 8.13.8
     repository: https://kubernetes-charts.storage.googleapis.com/
-    condition: prometheus-operator.enabled
+    condition: fluentd.metrics.enabled
```

Comment:
I believe we should use both conditions here, because now there is no way to disable operator without disabling metrics. Am I right?

Reply:
This should check for `sumologic.metrics.enabled` and `prometheus-operator.enabled`